diff --git a/node_modules/is-typed-array/index.js b/node_modules/is-typed-array/index.js
index a769ced..107a202 100644
--- a/node_modules/is-typed-array/index.js
+++ b/node_modules/is-typed-array/index.js
@@ -6,7 +6,7 @@ var callBound = require('call-bind/callBound');
 
 var $toString = callBound('Object.prototype.toString');
 var hasSymbols = require('has-symbols')();
-var hasToStringTag = hasSymbols && typeof Symbol.toStringTag === 'symbol';
+var hasToStringTag = hasSymbols && typeof Symbol.toStringTag === 'symbol' && Symbol.toStringTag in Float32Array;
 
 var typedArrays = availableTypedArrays();
 
